daphne_worker/src/config.rs

```diff
@@ -300,8 +301,16 @@ impl<D> DaphneWorkerConfig<D> {
             None => None,
         };
 
+        let url = req.url()?;
+        let path = url.path();
+        let mut router: Router<bool> = Router::new();
+        router.insert("/:version/*remaining", true).unwrap();
+        let url_match = router.at(path).unwrap();
+        let version = url_match.params.get("version").unwrap();
```

Comment:
Which of these unwrap()'s do you believe is safe? If the version prefix is missing or malformed, we'll want to handle this gracefully.

Reply:
The third one should be handled so that if there is no version, then we return an error.